# goal
float64 exposure  # (seconds) Must be greater than 0 or previous exposure will be used
---
# result
bool point_cloud_created
---
# feedback
